src/js/edu-benefits/1990e/config/form.js

```diff
@@ -6,8 +6,9 @@ import {
 import fullSchema1990e from 'vets-json-schema/dist/transfer-benefits-schema.json';
 
 import * as currentOrPastDate from '../../../common/schemaform/definitions/currentOrPastDate';
-import * as fullName from '../../../common/schemaform/definitions/fullName';
-import * as ssn from '../../../common/schemaform/definitions/ssn';
+import * as fullNameCommon from '../../../common/schemaform/definitions/fullName';
```

Comment:
I actually thought about that, but I wasn't sure how to handle it because I'm actually using the back-end schema for the regular schema (https://github.com/department-of-veterans-affairs/vets-website/blob/0056e270a153e62ade920213441b4eea4164b346/src/js/edu-benefits/1990e/config/form.js#L141) and the common schema for the uiSchema (https://github.com/department-of-veterans-affairs/vets-website/blob/0056e270a153e62ade920213441b4eea4164b346/src/js/edu-benefits/1990e/config/form.js#L124). I thought it would be important to use the same schema that the back-end uses to actually validate the fields. But I wasn't sure where we came out on that during the call yesterday. If you think I should use the common schema for the regular schema I can make that change!